bin/node/src/chain_spec.rs

```diff
@@ -129,37 +118,49 @@ pub struct ChainParams {
     #[structopt(long, default_value = "account_secret")]
     pub account_key_file: String,
 
+    // The length of a session
     #[structopt(long)]
     pub session_period: Option<u32>,
 
+    /// Time interval (in milliseconds) between blocks
     #[structopt(long)]
     pub millisecs_per_block: Option<u64>,
 
+    /// Chain name
     #[structopt(long, default_value = "Aleph Zero Development")]
     pub chain_name: String,
 
+    /// Token symbol
     #[structopt(long, default_value = "DZERO")]
     pub token_symbol: String,
 
-    /// Pass the AccountIds of authorities forming the committe at the genesis
-    ///
-    /// Expects a delimited collection of AccountIds
-    #[structopt(long, require_delimiter = true, parse(from_str = parse_account_id), required_unless("dev-chain"))]
+    /// AccountIds of authorities forming the committe at the genesis (comma delimited)
```

Comment:
you can correct the typo `committee` right away

Reply:
done